target/linux/brcm63xx/dts/dsl-2750u-c1.dts

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "bcm6328.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "D-Link DSL-2750u rev C1";
+	compatible = "d-link,dsl-2750u-c1", "brcm,bcm6328";
+
```

Comment:
@naseef  Could you please add LEDs aliases as suggested by [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist)
> 4. If it's possible try to dedicate some of the LEDs for system status indication in [diag.sh](https://github.com/openwrt/openwrt/blob/master/package/base-files/files/etc/diag.sh) ([example](https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9533_tplink_tl-wr841-v11.dts#L13))
https://github.com/openwrt/openwrt/blob/fe7d965ea95e78905328fe5425c8e90e3bf11e58/target/linux/ath79/dts/qca9533_tplink_tl-wr841-v11.dts#L13-L18

Reply:
Well it seems like is still not using generic diag.sh, so you can probably skip this one.